autoprep/utils/sampling.py

```diff
@@ -0,0 +1,10 @@
+from pandas import DataFrame
+from pandas import read_csv
+from random import sample
+
+
+def csv_sampling(file: str, size: int) -> DataFrame:
```

Comment:
`csv_sampling` should take offset or range which directs range of raw index number